client/cz-milestones-dash.html

```diff
@@ -131,17 +131,16 @@
       usersChanged: function(users) {
         // TODO: this probably only works for all-or-nothing changes to users right now.
         users.forEach(function(user, idx) {
-          registerSource('cz-issues', {owner: user.email}, function(data) {
-            for (var i = 0; i < data.length; i++) {
-              for (var j = 0; j < data[i].labels.length; j++) {
-                var label = data[i].labels[j];
+          registerSource('cz-monorail-issues', {owner: user.email}, function(issues) {
+            for (var issue of issues) {
+              for (var label of issue.labels) {
                 if (label.substring(0, 2).toLowerCase() == 'm-') {
```

Comment:
M doesn't really make sense for issues other than monorail, nor is it used by any other dash. I'd rather leave this behaviour scoped to this particular dash.